react/ProductCustomizer.js

```diff
@@ -1,54 +1,180 @@
+  * @return void
+  */
+  calculateTotal = () => {
+    let currentTotal = 0
+    const selectedVariations = this.state.selectedVariations
+
+    if (selectedVariations.length === 0) {
+      return 0
+    }
+
+    for (const selectedVariation of selectedVariations) {
+      currentTotal += selectedVariation.price
+    }
+
+    this.setState({ total: currentTotal })
+  }
+
+  /**
+  * handleOpenModal
+  * Show the product modal
+  * @return void
+  */
+  handleOpenModal = () => {
     this.setState({ isModalOpen: true })
   }
 
-  handleCloseModal() {
+  /**
+  * handleCloseModal
+  * Close the product modal
+  * @return void
+  */
+  handleCloseModal = () => {
     this.setState({ isModalOpen: false })
   }
 
+  handleSubmit = e => {
+    console.log(this.state.selectedVariations)
+    e.preventDefault()
+
+    // Checl Lists validations
+    // Add product to cart
+  }
+
   render() {
+    const {
+      variations,
+      selectionsValidated,
+      onHandleUpdateAmount,
+    } = this.state
+
+    const { enableChangeToppings } = this.props
+
     return (
       <div>
         <Button onClick={this.handleOpenModal}>Customize your product</Button>
 
         <Modal
-          top
-          isMobile="true"
+          centered
           isOpen={this.state.isModalOpen}
           onClose={this.handleCloseModal}
         >
           <div className={'flex-ns h-100-ns'}>
-            <div className={'w-100 w-third-ns bg-light-gray bb b--light-gray flex-ns items-center-ns  h-100-ns'}>
-              <h1 className={`vtex-product-customizer__title tc f4 fw5 absolute ma0 pa5 w-100 bg-black-40 white ${!isMobile ? 'dn' : ''}`}>Create Your Own</h1>
+            <h1 className={`vtex-product-customizer__title tc f4 fw5 ma0 pa5 w-100 bg-black-40 white ${!isMobile ? 'dn' : ''}`}>Create Your Own</h1>
+            <div className={'w-100 w-third-ns flex-ns tc items-center-ns pa5 h-100-ns'}>
               <img
-                className={'vtex-product-customizer__image'}
-                alt="Create Your Own"
-                src="https://via.placeholder.com/670x500"
+                className={'vtex-product-customizer__image br3'}
+                alt="Product Customize Image"
+                src="https://via.placeholder.com/330x330"
               />
             </div>
             <div className={'w-100 w-two-thirds-ns flex-ns flex-column-ns'}>
-              <h1 className={`vtex-product-customizer__title fw5 ma0 f3 pa5 ${isMobile ? 'dn' : ''}`}>Create Your Own</h1>
-              <div className={'pb5-ns pt0-ns ph5-ns pa5 bb b--light-gray'}>
-                <p className={'ma0 fw3'}>Go freestyle! Simply choose your sauce and add any toppings.</p>
-              </div>
-              <AttachmentsList />
+              <form name="vtex-product-customizer-form" onSubmit={this.handleSubmit}>
+                <h1 className={`vtex-product-customizer__title fw5 ma0 f3 pa5 ${isMobile ? 'dn' : ''}`}>Create Your Own</h1>
```

Comment:
Tachyons already provides a way to hide html tags when you are in mobile mode. In this cases, for example, you can use dn and db-ns at the same time. I am pretty sure that this `isMobile` from window is not needed in this context.